src/main/java/com/znsio/teswiz/runner/DeviceSetup.java

```diff
@@ -8,11 +8,12 @@
 import com.znsio.teswiz.tools.cmd.CommandLineResponse;
 import org.apache.log4j.Logger;
 
-import java.io.File;
-import java.io.IOException;
+import java.io.*;
```

Comment:
Do we have more than 5 imports for java.io, if not could we explicitly mention them explicitly?

Reply:
Exactly 5 are present, it is handled by IDE itself